src/scripts/shell/platform-choice/index.js

```diff
@@ -0,0 +1 @@
+export * from './platform-choice.component';
```

Comment:
Bundles many imports into one export.  Called a bucket.  Makes importing elsewhere require fewer files for importing.